embedded-redis/src/main/java/com/playtika/test/redis/RedisStatusCheck.java

```diff
@@ -34,7 +34,7 @@
     @Override
     public String[] getCheckCommand() {
         return new String[] {
-                "bash", "/redis-health.sh"
+                "redis-cli", "-a", "$REDIS_PASSWORD", "ping"
```

Comment:
result is not asserted here.

Reply:
Result is asserted by AbstractCommandWaitStrategy: redis-cli will exit with status code "1" if will be unable to connect/execute command

Reply:
I'm ok if result content is not needed, and result code is enough, any doc that can confirm it?